src/main/java/io/electrum/vas/model/QrPayment.java

```diff
@@ -0,0 +1,100 @@
+package io.electrum.vas.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+
+/**
+ * @since v1.31.0
```

Comment:
Missed this in the last round, my bad

```suggestion
 * @since v3.31.0
```